starter-code/index.js

```diff
@@ -1,10 +1,43 @@
-var SortedList = function() {};
+var SortedList = function() {
+    this.items = [];
+    this.length = this.items.length;
+};
+
+SortedList.prototype.add = function(item) {
+    this.items.push(item);
+    this.items.sort(function(a,b){
```

Comment:
Now you know how to use nice ES6 syntax like this:
this.items.sort( (a, b) => a - b )

:)